docs/publish/failure-type-classification-with-the-testgrid-data-project-doc.md

```diff
@@ -0,0 +1,75 @@
+
+If this project is successful, we will develop a tool to automatically analyze the TestGrid data. This tool will perform failure type classification with the Testgrid data to address an existing manual process executed by subject matter experts. Using the tool, the developers can focus on real issues and become more productive. Furthermore, we will provide insights about overall statistics about failures so that test developers can improve on existing test suites.
+
+### A. Problem Statement
+
+Given a TestGrid, we want to classify/detect different failure patterns occurring over a certain period of time. In the later part, we will aggregate the results to conclude about the primary reasons behind failures for each release.
+
+### B. Checklist for project completion
+
+1. A notebooks that shows classification and analysis of different types of test failures on TestGrid data.
+
+2. Jupyterhub image to reproduce the results.
+
+3. Public blog explaining analysis and results.
+
+4. Results hosted for SME to review
+
+### C. Provide a solution in terms of human actions to confirm if the task is within the scope of automation through AI.
+
+Without AI and automation tooling, SME will need to go to TestGrid data of a particular release and look at the failures. An SME will determine if that failure is following any of the patterns that we have discussed in earlier sections. Based on detected patterns, an SME tries to determine the reason behind failure.
+
+### D. Outline a path to operationalization.
+
+Once we have notebooks ready, we will build a Notebook-based Pipeline using Elyra. The results will be stored in S3. We can then use Superset as our dashboard and visualization tool, which SME/developers can access and give feedback. Once SMEs are satisfied with the result, we can create PR to integrate our work directly into the TestGrid platform.
+
+## Resources
+
+### Data Sets
+
+* [https://testgrid.k8s.io/redhat-openshift-informing](https://testgrid.k8s.io/redhat-openshift-informing)
+
+Instruction regarding downloading data: [TestGrid_EDA.ipynb](https://github.com/aicoe-aiops/ocp-ci-analysis/blob/master/notebooks/TestGrid_EDA.ipynb)
+
+## How to Contribute / Provide feedback
+
+1. Github Repository: [https://github.com/aicoe-aiops/ocp-ci-analysis](https://github.com/aicoe-aiops/ocp-ci-analysis)
+
+2. You can open up a PR on the Git Repository highlighting the feature or issue, and we will address it.
+
+3. You can also reach out to sbadhe@redhat.com for any questions.
+
+## Weekly Meeting Updates
+
+*Keep track of ongoing meetings below.*
+
+### Year-Month-Day
+
+* Item 1
+
+* Item 2
```

Comment:
you can remove this section

Reply:
Hi @MichaelClifford , 
I am planning to remove this section except the dataset part. 
what do you say?